.github/workflows/ci.yml

```diff
@@ -116,6 +116,10 @@ jobs:
             args: --all-targets --all-features
           - command: test
             args: --all-targets --all-features
+          - command: test
+            args: --all-targets --no-default-features
+          - command: build
```

Comment:
I'm not sure you need this build step -- how could cargo test something it has not built ?

If you ask it to test without default features it will be forced to build the extra code that was enabled.

So doing a build after that is redundant.